config/terrains.json

```diff
@@ -117,6 +126,7 @@
 		"code" : "rc",
 		"battleFields" : ["rocklands"],
 		"transitionRequired" : true,
-		"terrainViewPatterns" : "rock"
+		"terrainViewPatterns" : "rock",
+		"horseSoundId" : 9
```

Comment:
Interesting why do we have this change here

Reply:
In general it's because of the bug discovered by @Macron1Robot: it was wrong sounds.
If you are speaking about rock in particular - just for unification